analyticsclient/users.py

```diff
@@ -0,0 +1,54 @@
+import analyticsclient.constants.data_format as DF
+
+
+class UserList(object):
+    """ Fetch list of users that are available forser-related analytics. """
+
+    def __init__(self, client):
+        """
+        Initialize the API client.
+
+        Arguments:
+
+            client (analyticsclient.client.Client): The client to use to access remote resources.
+
+        """
+        self.client = client
+
+    def list_users(self, page=1, limit=100, data_format=DF.JSON):
+        """
+        Get the list of users.
+
+        Arguments:
+            page (int): Which page of results to return
+            limit (int): The maximum number of users to include on each page
```

Comment:
The `data_format` parameter seems to be documented on all other endpoints, it would make sense to document it here, too.